vip_hci/fits/fits.py

```diff
@@ -90,7 +91,7 @@ def open_adicube(fitsfilename, verbose=True):
     
     Parameters
     ----------
-    fitsfilename : string
+    fitsfilename : string or Path
         Name of the fits file.
```

Comment:
I think it's worth explaining here the pathlib.Path option ;)

Reply:
Hi @r4lv, I think you missed this comment. It's just changing the description of the fitsfilename. Same for open_fits

Reply:
I mean to extend the description of the argument: Name of the fits file or .... object.